app/helpers/concerns/foreman_remote_execution/hosts_helper_extensions.rb

```diff
@@ -8,11 +8,27 @@ module HostsHelperExtensions
     end
 
     def multiple_actions_with_remote_execution
-      multiple_actions_without_remote_execution + [[_('Run Job'), new_job_invocation_path, false]]
+      multiple_actions_without_remote_execution + [ [_('Schedule Job'), new_job_invocation_path, false] ]
+    end
+
+    def schedule_job_multi_button(*args)
+      host_features = RemoteExecutionFeature.where(:host_action_button => true).map do |feature|
+        link_to(_('%s') % feature.name, job_invocations_path(:host_ids => [args.first.id], :feature => feature.label), :method => :post)
+      end
+
+      if host_features.present?
+        action_buttons(schedule_job_button(*args), *host_features)
+      else
+        schedule_job_button(*args)
+      end
+    end
+
+    def schedule_job_button(*args)
+      link_to(_('Schedule Job'), new_job_invocation_path(:host_ids => [args.first.id]), :id => :run_button, :class => 'btn btn-default')
```

Comment:
What about "Schedule Remote Job"? My concern is the Job name itself is not well descriptive. For some reason, comparing `Run Job` and `Schedule Job`, the `Run Job` felt more like it has something to do with Rex. The `schedule` seems more generic to me, and I feel specifying what kind of job we mean might help with user understanding. On the other hand, I'm not going to bikeshed on this.